near-mock-vm/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "near-mock-vm"
+version = "0.1.0"
+authors = ["Willem Wyndham <willem@cs.umd.edu>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["console_error_panic_hook"]
+
+[dependencies]
+serde_json = "1.0"
+base64 = "0.12"
+wasm-bindgen = "0.2.60"
+serde = { version = "1.0", features = ["derive"] }
+serde-wasm-bindgen = "0.1.3"
+js-sys = "0.3.37"
+near-sdk = { git="https://github.com/near/near-sdk-rs", branch = "add_wasm_mock_vm" }
+near-vm-errors = "2.2.0"
+near-vm-logic = "2.2.0"
+pwasm-utils = "0.12.0"
+parity-wasm = "0.41.0"
+wasmparser = "0.51.4"
+
+
+# The `console_error_panic_hook` crate provides better debugging of panics by
+# logging them with `console.error`. This is great for development, but requires
+# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
+# code size when deploying.
+console_error_panic_hook = { version = "0.1.1", optional = true }
+
+# `wee_alloc` is a tiny allocator for wasm that is only ~1K in code size
+# compared to the default allocator's ~10K. It is slower than the default
+# allocator, however.
+#
+# Unfortunately, `wee_alloc` requires nightly Rust when targeting wasm for now.
+#wee_alloc = { version = "0.4.2", optional = true }
```

Comment:
I just turned it into a feature.  I don't think that people would want it one unless it was going to be used in a browser context, since it's faster without it.